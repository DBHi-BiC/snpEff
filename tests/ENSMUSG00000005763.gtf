1	protein_coding	exon	167718812	167719017	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167718960	167719017	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	start_codon	167718960	167718962	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	exon	167785297	167785400	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167785297	167785400	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167786531	167786587	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167786531	167786587	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167787547	167787627	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167787547	167787627	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167788574	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167788574	167788609	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167789350	167789406	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167789920	167789955	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	exon	167799149	167801284	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	CDS	167799149	167799250	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-001"; protein_id "ENSMUSP00000124297";
1	protein_coding	stop_codon	167799251	167799253	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161971"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-001";
1	protein_coding	exon	167718820	167719017	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167718960	167719017	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	start_codon	167718960	167718962	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	exon	167785297	167785400	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167785297	167785400	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167786531	167786587	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167786531	167786587	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167787547	167787627	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167787547	167787627	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167788574	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167788574	167788609	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167789350	167789406	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167789920	167789955	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	exon	167791135	167792243	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	CDS	167791135	167791197	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-004"; protein_id "ENSMUSP00000005907";
1	protein_coding	stop_codon	167791198	167791200	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000005907"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-004";
1	protein_coding	exon	167718854	167719017	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167718960	167719017	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	start_codon	167718960	167718962	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	exon	167785297	167785400	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167785297	167785400	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167786531	167786587	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167786531	167786587	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167787547	167787627	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167787547	167787627	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167788574	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167788574	167788609	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167789350	167789406	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167789920	167789955	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	exon	167796786	167797565	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	CDS	167796786	167796974	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-003"; protein_id "ENSMUSP00000027849";
1	protein_coding	stop_codon	167796975	167796977	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000027849"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-003";
1	protein_coding	exon	167718896	167719017	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167718960	167719017	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	start_codon	167718960	167718962	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	exon	167785297	167785400	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167785297	167785400	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167786531	167786587	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167786531	167786587	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167787547	167787627	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167787547	167787627	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167788574	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167788574	167788609	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167789350	167789406	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167789920	167789955	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167796786	167796879	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167796786	167796879	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	exon	167799149	167800386	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "9"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	CDS	167799149	167799192	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "9"; gene_name "Cd247"; transcript_name "Cd247-002"; protein_id "ENSMUSP00000124299";
1	protein_coding	stop_codon	167799193	167799195	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000161559"; exon_number "9"; gene_name "Cd247"; transcript_name "Cd247-002";
1	protein_coding	exon	167718896	167719017	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167718960	167719017	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	start_codon	167718960	167718962	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	exon	167785297	167785400	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167785297	167785400	.	+	2	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167786531	167786587	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167786531	167786587	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167787547	167787627	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167787547	167787627	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167788574	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167788574	167788609	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "5"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167789350	167789406	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "6"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167789920	167789955	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "7"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	exon	167796786	167797029	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	CDS	167796786	167796974	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-005"; protein_id "ENSMUSP00000083165";
1	protein_coding	stop_codon	167796975	167796977	.	+	0	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "8"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	exon	167799149	167799225	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000086002"; exon_number "9"; gene_name "Cd247"; transcript_name "Cd247-005";
1	protein_coding	exon	167788347	167788609	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000159070"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-007";
1	protein_coding	exon	167789350	167789406	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000159070"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-007";
1	protein_coding	exon	167789920	167789955	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000159070"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-007";
1	protein_coding	exon	167791135	167791604	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000159070"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-007";
1	protein_coding	exon	167794786	167794871	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000162411"; exon_number "1"; gene_name "Cd247"; transcript_name "Cd247-006";
1	protein_coding	exon	167796786	167796879	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000162411"; exon_number "2"; gene_name "Cd247"; transcript_name "Cd247-006";
1	protein_coding	exon	167799149	167799332	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000162411"; exon_number "3"; gene_name "Cd247"; transcript_name "Cd247-006";
1	protein_coding	exon	167806951	167807407	.	+	.	 gene_id "ENSMUSG00000005763"; transcript_id "ENSMUST00000162411"; exon_number "4"; gene_name "Cd247"; transcript_name "Cd247-006";
