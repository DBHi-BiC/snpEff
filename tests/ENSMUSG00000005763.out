1	167789347	A	C	+	0	0	INTRON
1	167789348	A	C	+	0	0	SPLICE_SITE_ACCEPTOR
1	167789349	A	C	+	0	0	SPLICE_SITE_ACCEPTOR
